### PR TITLE
fix: Cleanup AboutUs page

### LINKED ISSUE #28: Clean up AboutMe.md
Post merge clean up required. Some tasks include:
- [x] Standardizing portfolio description
- [x] Removing redundant info in`AboutUs.md`

### PR DESCRIPTION
**Pull request by jayjay19630**
Resolves #28

### CODE REVIEW

File: docs/AboutUs.md [hunk 1]
```diff
@@ -32,45 +33,16 @@ You can reach us at our individual emails below!
   * Integration
```
Comment (zaidansani) [inline docs/AboutUs.md:33]: I think we can change this to match the rest of the formats:
```
Roles:
Responsibilities:
```

**Review by zaidansani — CHANGES_REQUESTED** [review]
LGTM - just some minor changes would be good

File: docs/AboutUs.md (hunk 1 above)
Comment (jayjay19630) [reply docs/AboutUs.md]: Sounds good, I have just made the necessary changes!

**Review by zaidansani — APPROVED** [review]
LGTM.

**Pull request merged**